build/packages.targets

```diff
@@ -29,6 +29,7 @@
         <PackageReference Update="Microsoft.Build.Locator" Version="1.4.1" />
         <PackageReference Update="Microsoft.Build.Tasks.Core" Version="$(MicrosoftBuildPackageVersion)" />
         <PackageReference Update="Microsoft.Build.Utilities.Core" Version="$(MicrosoftBuildPackageVersion)" />
+        <PackageReference Update="Microsoft.CodeAnalysis.BannedApiAnalyzers" Version="3.0.0" />
```

Comment:
Can we use `3.3.2` version which is the latest stable version instead of `3.0.0` version? https://www.nuget.org/packages/Microsoft.CodeAnalysis.BannedApiAnalyzers.